pkg/gofr/http/multipart_file_bind_test.go

```diff
@@ -54,3 +55,129 @@ func TestGetFieldName(t *testing.T) {
 		})
 	}
 }
+
+type testValue struct {
+	kind  reflect.Kind
+	value interface{}
+}
+
+func Test_SetFieldValue_Success(t *testing.T) {
+	testCases := []struct {
+		desc      string
+		data      string
+		expected  bool
+		valueType testValue
+	}{
+		{"String", "test", true, testValue{reflect.String, "string"}},
+		{"Int", "10", true, testValue{reflect.Int, 0}},
+		{"Uint", "10", true, testValue{reflect.Uint16, uint16(10)}},
+		{"Float64", "3.14", true, testValue{reflect.Float64, 0.0}},
+		{"Bool", "true", true, testValue{reflect.Bool, false}},
+		{"Slice", "1,2,3,4,5", true, testValue{reflect.Slice, []int{}}},
+		{"Array", "1,2,3,4,5", true, testValue{reflect.Array, [5]int{}}},
+		{"Struct", `{"name": "John", "age": 30}`, true, testValue{reflect.Struct, struct {
+			Name string `json:"name"`
+			Age  int    `json:"age"`
+		}{}}},
+		{"Interface", "test interface", true, testValue{reflect.Interface, new(any)}},
+	}
+
+	for _, tc := range testCases {
+		f := &formData{}
+		val := reflect.New(reflect.TypeOf(tc.valueType.value)).Elem()
+
+		set, err := f.setFieldValue(val, tc.data)
+		if err != nil {
+			t.Errorf("Unexpected error for value kind %v and data %q: %v", val.Kind(), tc.data, err)
+		}
```

Comment:
You are already using testify in the file

```suggestion
		require.NoErrorf(t, err, "Unexpected error for value kind %v and data %q", val.Kind(), tc.data)
```

Reply:
sure updated